### PR TITLE
Clarify that UTF-8 is the default charset #462

### LINKED ISSUE #462: Clarify character encoding
The API doesn't say anything about character encoding and I'm not sure all clients send/parse the corresponding headers. Should we say something that if nothing is negotiated with HTTP content negotiation (`Charset` / `Accept-Charset`) that the default character encoding for the API implementations is UTF-8?

Origin: https://github.com/Open-EO/openeo-r-client/issues/132

### PR DESCRIPTION
**Pull request by m-mohr**
Fixes #462

- Explicitly mention the use of HTTP content negotiation
- Clarify that the default charset is UTF-8 [#462](https://github.com/Open-EO/openeo-api/issues/462)

### CODE REVIEW

**Review by dthiex — APPROVED** [review]

File: openapi.yaml [hunk 1]
```diff
@@ -27,9 +27,16 @@ info:
...
 
-    ## JSON
+    The openEO API makes use of [HTTP Content Negotiation](https://www.rfc-editor.org/rfc/rfc9110.html#name-content-negotiation),
+    including, but not limited to, the request headers `Accept`, `Accept-Charset` and `Accept-Language`.
 
-    The API uses JSON for request and response bodies whenever feasible. Services use JSON as the default encoding. Other encodings can be requested using [Content Negotiation](https://www.w3.org/Protocols/rfc2616/rfc2616-sec12.html). Clients and servers MUST NOT rely on the order in which properties appears in JSON. Collections usually don't include nested JSON objects if those information can be requested from the individual resources.
+    ### JSON
+
+    The API uses JSON for request and response bodies whenever feasible. Services use JSON as the default encoding. Other encodings can be requested using HTTP Content Negotiation ([`Accept` header](https://www.rfc-editor.org/rfc/rfc9110.html#name-accept)). Clients and servers MUST NOT rely on the order in which properties appears in JSON. Collections usually don't include nested JSON objects if those information can be requested from the individual resources.
```
Comment (SerRichard) [inline openapi.yaml:35]: The last sentence seems ambiguous, I'm not sure what it means? Also, typo on 'properties appears' -> to 'properties appear'!

**Review by SerRichard — COMMENTED** [review]
Only one comment on line 35, unsure what the last sentence really means, and also just highlighting a typo.

File: openapi.yaml (hunk 1 above)
Comment (m-mohr) [reply openapi.yaml:35]: No, it was meant as it is (but maybe the additional comma helps?) :-)

Comment (SerRichard) [reply openapi.yaml:35]: ahhh yes I understand, sorry for the slowness!

**Pull request merged**